src/preferences.jl

```diff
@@ -0,0 +1,57 @@
+
+"""
+```julia
+save_config!(config::CthulhuConfig=CONFIG, force=true)
+```
+Save a Cthulhu.jl configuration `config` (by default, `Cthulhu.CONFIG`) to your
+`LocalPreferences.toml` file using Preferences.jl.
+
+The saved preferences will be automatically loaded next time you `using Cthulhu`
+
+## Examples
+```julia
+julia> using Cthulhu
+
+julia> Cthulhu.CONFIG.enable_highlighter = true
+true
+
+julia> Cthulhu.CONFIG.debuginfo = :none     # Customize some defaults
+:none
+
+julia> Cthulhu.save_config!(Cthulhu.CONFIG) # Will be automatically read next time you `using Cthulhu`
+```
+"""
+function save_config!(config::CthulhuConfig=CONFIG, force=true)
+    cthulhu_id = UUID("f68482b8-f384-11e8-15f7-abe071a5a75f")
+    set_preferences!(cthulhu_id,
```

Comment:
```suggestion
    @set_preferences!(
```

Reply:
Ah yeah, I couldn't figure out how to get the macros when doing the "use without adding as dependency" trick, but we're not doing that any more so might as well go for the macros!